company-profiles/product-hackers.md

```diff
@@ -0,0 +1,33 @@
+# Product Hackers
+
+## Company blurb
+
+Product Hackers is an early company based in Madrid which develop cool software and technology, mainly web and APP. We offer consultancy services, **growth** services, _which are our core_, and develop in-house products to sell them worldwide.
```

Comment:
Typo here: "Product Hackers is an early company based in Madrid which develop[s]" (missing `s`)